src/Public/Routes.ps1

```diff
@@ -2166,10 +2166,10 @@ function Add-PodePage {
 
                 # invoke the function (optional splat data)
                 if (Test-PodeIsEmpty $data) {
-                    $result = (. $script)
+                    $result = & $script
                 }
                 else {
-                    $result = (. $script @data)
+                    $result = & $script @data
                 }
```

Comment:
I just tested this locally myself, and even with the switch to `&` I still get the issue.

I changed the block to use `Invoke-PodeScriptBlock` instead and that seems to have fixed it on myside - and sped it up as well:
```poweshell
if (Test-PodeIsEmpty $data) {
    $result = Invoke-PodeScriptBlock -ScriptBlock $script -Return
}
else {
    $result = Invoke-PodeScriptBlock -ScriptBlock $script -Arguments $data -Return
}
```

Reply:
with & was working for me.
but I'm happy to change to `Invoke-PodeScriptBlock`